Classes/Controller/AbstractFluxController.php

```diff
@@ -12,14 +12,15 @@
 use FluidTYPO3\Flux\Service\WorkspacesAwareRecordService;
 use FluidTYPO3\Flux\Utility\ExtensionNamingUtility;
 use FluidTYPO3\Flux\Utility\RecursiveArrayUtility;
-use FluidTYPO3\Flux\View\ExposedTemplateView;
-use FluidTYPO3\Flux\ViewHelpers\Outlet\FormViewHelper;
+use FluidTYPO3\Flux\ViewHelpers\FormViewHelper;
 use TYPO3\CMS\Core\Utility\GeneralUtility;
 use TYPO3\CMS\Extbase\Configuration\ConfigurationManagerInterface;
 use TYPO3\CMS\Extbase\Mvc\Controller\ActionController;
 use TYPO3\CMS\Extbase\Mvc\Exception\StopActionException;
 use TYPO3\CMS\Extbase\Mvc\View\ViewInterface;
 use TYPO3\CMS\Extbase\Mvc\Web\Response;
+use TYPO3Fluid\Fluid\Core\Rendering\RenderingContextInterface;
+use function get_class;
```

Comment:
Yep, I may do a full conversion to import global functions, there are thousands of opcode calls to be saved here.